src/txmempool.cpp

```diff
@@ -55,10 +55,9 @@ double
 CTxMemPoolEntry::GetPriority(unsigned int currentHeight) const
```

Comment:
@MarcoFalke What should be dropped? Sorry, didn't catch that :-)

Do you mean that the changes to `src/txmempool.cpp` should be excluded from this commit?